### PR TITLE
Fix the documentation and some small issues 

### PR DESCRIPTION
**Pull request by stelmo**
Update the names of the modification functions to be more consistent
Extend the notebooks to showcase them

### CODE REVIEW

**Comment by cylon-x** [comment]
:red_square: &nbsp;Commit 781ef4a2 requires formatting!

Required formatting changes summary:
```
 docs/src/notebooks/2_finding_balance.jl | 6 +++++-
 test/analysis/loopless.jl               | 6 +++++-
 2 files changed, 10 insertions(+), 2 deletions(-)
```

**Comment by cylon-x** [comment]
:red_square: &nbsp;Commit c33aadff requires formatting!

Required formatting changes summary:
```
 docs/src/notebooks/2_finding_balance.jl | 10 ++++++----
 test/analysis/loopless.jl               |  6 +++++-
 2 files changed, 11 insertions(+), 5 deletions(-)
```

**Comment by cylon-x** [comment]
:green_circle: &nbsp;Commit 0d0714f7 is formatted properly.

**Comment by cylon-x** [comment]
:green_circle: &nbsp;Commit 138134cf is formatted properly.

**Comment by cylon-x** [comment]
:green_circle: &nbsp;Commit 138134cf is formatted properly.

**Comment by cylon-x** [comment]
:green_circle: &nbsp;Commit 423144f9 is formatted properly.

File: docs/src/notebooks/2_finding_balance.jl [hunk 1]
```diff
@@ -52,15 +52,16 @@ vec_soln = flux_balance_analysis_vec(model, Tulip.Optimizer)
 #
 dict_soln = flux_balance_analysis_dict(model, Tulip.Optimizer)
 
-# ## Modifications
+# ## Extended FBA through modifications
```
Comment (exaexa) [inline docs/src/notebooks/2_finding_balance.jl:55]: "Extending FBA with modifications"

Comment (exaexa) [reply docs/src/notebooks/2_finding_balance.jl:55]: ...I'll edit this in github myself lol.

**Review by exaexa — APPROVED** [review]
Please check my reword, then fftm

**Comment by cylon-x** [comment]
:green_circle: &nbsp;Commit b9be0759 is formatted properly.

**Comment by cylon-x** [comment]
:green_circle: &nbsp;Commit 9eeef281 is formatted properly.

**Pull request merged**